packages/synapse-react-client/src/components/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/ManagedACTAccessRequirementItem.tsx

```diff
@@ -17,7 +16,6 @@ import {
 import { RequirementItemStatus } from '../AccessApprovalCheckMark'
 
 export type ManagedACTAccessRequirementItemProps = {
-  entityId: string
```

Comment:
Unused

Reply:
Oh I see!

Reply:
We already have the AR, so we only need it in the other case because we will send the user to the Synapse.org page with this legacy AR.  Got it